packages/jest-worker/src/workers/messageParent.ts

```diff
@@ -7,31 +7,33 @@
 
 import {PARENT_MESSAGE_CUSTOM} from '../types';
 
-const isWorkerThread = () => {
+const isWorkerThread: boolean = (() => {
   try {
     // `Require` here to support Node v10
-    const {isMainThread, parentPort} = require('worker_threads');
-    return !isMainThread && parentPort;
+    const {
+      isMainThread,
+      parentPort,
+    } = require('worker_threads') as typeof import('worker_threads');
+    return !isMainThread && parentPort != null;
   } catch {
     return false;
   }
-};
+})();
 
-const messageParent = (
+export default function messageParent(
```

Comment:
should ignore whitespace here - diff is removed `try-catch` and added type information

https://github.com/facebook/jest/pull/10984/files?w=1